jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/DefaultProjectAccountSettingsManager.kt

```diff
@@ -25,6 +26,7 @@ data class ConnectionSettingsState(
 class DefaultProjectAccountSettingsManager(private val project: Project) : ProjectAccountSettingsManager(project),
     PersistentStateComponent<ConnectionSettingsState> {
     override fun getState(): ConnectionSettingsState = ConnectionSettingsState(
+        activePartition = selectedPartition?.id,
```

Comment:
Do we need this? Or can we infer it from active region?

Reply:
Probably cleaner to infer, I don't think it's worth persisting any transient state resulting from the user switching between partitions.